cfme/common/host_views.py

```diff
@@ -208,6 +209,10 @@ def entity_class(self):
         return HostEntity
 
 
+class HostPrintView(BaseLoggedInPage):
+    pass
```

Comment:
Is there anything we can check here?

Reply:
Any suggestions? This is a pretty unique page. I kinda skirted around adding a locator or widgets and functions. The manual test simply made sure the print window appeared without testing print or export.  I'll see what I can get working.

Reply:
As per our discussion in sync, I believe what we have will work. This in not an actual 'print or export' test.